src/DistributedLock.Tests/Tests/Etcd/EtcdLockTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Medallion.Threading.Etcd;
+using NUnit.Framework;
+
+namespace Medallion.Threading.Tests.Etcd;
+
+[Category("CI")]
```

Comment:
Do these tests actually run in the (very simple) CI system? If not, let's remove this category